.circleci/config.yml

```diff
@@ -28,9 +28,9 @@ store-log-artifacts: &store-log-artifacts
     path: log
     destination: log
 
-npm-cache-key: &npm-cache-key node-v10-{{ checksum "yarn.lock" }}-5
-bundle-cache-key: &bundle-cache-key v2-bundler-gems-{{ .Environment.DB }}-{{ arch }}-{{ .Branch }}-{{ checksum "Gemfile.lock" }}
-assets-cache-key: &assets-cache-key v1-asset-cache-{{ checksum "tmp/assets_related_checksums" }}
+npm-cache-key: &npm-cache-key porta-{{ .Environment.CACHE_VERSION }}-node-v10-{{ checksum "yarn.lock" }}
```

Comment:
Do you know why `node-v10` is there? do we need it?

Reply:
No, I don't think it has any usage. Removing it only invalidates the cache, nothing else. I assume `yarn.lock` will be different for different node versions, so different node versions will never share the cache. I'm gonna remove it.

Reply:
Yes, I think it is confusing (like - why v10? we're on v14 now). Thanks!